backend/src/test/java/de/bund/digitalservice/ris/norms/integration/adapter/output/database/DBServiceIntegrationTest.java

```diff
@@ -274,7 +274,11 @@ void itFindsAnnouncementOnDB() {
     );
 
     // Then
-    assertThat(announcementOptional).isPresent().contains(announcement);
+    assertThat(announcementOptional)
+      .get()
+      .usingRecursiveComparison()
+      .ignoringFields("importTimestamp")
```

Comment:
Even if we are not setting the timestamp in the application and letting the db create that value I would include a "checking" to some level in an integration test, like maybe just checking that the value is not empty? Or even checking if the value is within the range we would expact (minutes wise?)